From 966cf3fe1cd3654b023bb51d79a334902264c344 Mon Sep 17 00:00:00 2001
From: Martin Jansa <martin.jansa@lge.com>
Date: Fri, 6 Sep 2019 06:56:00 -0700
Subject: [PATCH] CMakeLists.txt: install libtatlas.so in regular libdir
 instead of /usr/local/atlas/lib

:Release Notes:
With newer meta-updater this only file in /usr/local breaks
do_image_ostree.

:Detailed Notes:
With:
https://github.com/advancedtelematic/meta-updater/commit/bc6c9f098442a2c0bf4a7a40ba50c9ee5fe40cc1#diff-e2abc18e16bca9de6396cc6ab17ac27f
it fails with:
WARNING: webos-image-devel-1.0-r2 do_image: Android repo tool not found; manifest not copied.
WARNING: webos-image-devel-1.0-r2 do_image_ostree: Data in /opt directory is not preserved by OSTree. Consider moving it under /usr
WARNING: webos-image-devel-1.0-r2 do_image_ostree: Data in /mnt directory is not preserved by OSTree. Consider moving it under /usr
ERROR: webos-image-devel-1.0-r2 do_image_ostree: Data in /usr/local directory is not preserved by OSTree.
ERROR: webos-image-devel-1.0-r2 do_image_ostree: Function failed: do_image_ostree (log file is located at /jenkins/mjansa/build-webos-thud/BUILD/work/raspberrypi3-webos-linux-gnueabi/webos-image-devel/1.0-r2/temp/log.do_image_ostree.16291)
ERROR: Logfile of failure stored in: /jenkins/mjansa/build-webos-thud/BUILD/work/raspberrypi3-webos-linux-gnueabi/webos-image-devel/1.0-r2/temp/log.do_image_ostree.16291

:Testing Performed:

:QA Notes:

:Issues Addressed:
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 13f6a65..4747f00 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -126,7 +126,7 @@ install(
 
 install(
     PROGRAMS ${CMAKE_SOURCE_DIR}/lib/libtatlas.so
-    DESTINATION /usr/local/atlas/lib
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}
 )
 
 file(GLOB_RECURSE PUBLIC_HEADERS ${CMAKE_SOURCE_DIR}/include/*.h)
-- 
2.17.1

